docs/guides/running-clustered-experiments.md

```diff
@@ -0,0 +1,130 @@
+---
+sidebar_position: 2
```

Comment:
Do you have to update the sidebar_position for other articles?

Reply:
I just removed this. I had copied it over from another template https://github.com/Eppo-exp/eppo-docs/commit/51f012a67d16f13f814bed0c7bc9a9bcdc2fa43c